tests/testthat/test-a11y_panel.R

```diff
@@ -0,0 +1,152 @@
+test_that("Accessibility dates are required", {
+  expect_no_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26th November 2024",
+      date_prepared = "28th November 2024",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    )
+  )
+  expect_no_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26 November 2024",
+      date_prepared = "28/11/2024",
+      date_reviewed = "12-03-2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    )
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_prepared = "28th November 2024",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "argument \"date_tested\" is missing, with no default"
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26/11/24",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "argument \"date_prepared\" is missing, with no default"
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26th November 2024",
+      date_prepared = "28th November 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "argument \"date_reviewed\" is missing, with no default"
+  )
+  expect_error(
+    a11y_panel(
+      "DfE Shiny template",
+      "https://department-for-education.shinyapps.io/dfe-shiny-template",
+      date_tested = "26th November 2025",
+      date_prepared = "28th November 2024",
+      date_reviewed = "12th March 2024",
+      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
+    ),
+    "26th November 2025 is in the future."
```

Comment:
I think it's worth going a bit further into the future so this test doesn't break next year?

```suggestion
      date_tested = "1st January 3000",
      date_prepared = "28th November 2024",
      date_reviewed = "12th March 2024",
      repo_url = "https://github.com/dfe-analytical-services/shiny-template"
    ),
    "1st January 3000 is in the future."
```

Reply:
Yeah, good point, I didn't think that through properly did I! Updated beyond my lifetime now at least!